yb-voyager/src/dbzm/dbzm.go

```diff
@@ -9,25 +9,41 @@ import (
 	"time"
 
 	"github.com/tebeka/atexit"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 	"gopkg.in/natefinch/lumberjack.v2"
 
 	log "github.com/sirupsen/logrus"
 )
 
 var DEBEZIUM_DIST_DIR, DEBEZIUM_CONF_FILEPATH string
 
+const DEBEZIUM_BREW_VERSION = "2.2.0-voyager-1.3"
```

Comment:
Add a step in our release checklist to update this version on each voyager release. Also add a comment here as well.
Make the `2.2.0-voyager-1.3` the DEBEZIUM_VERSION _everywhere_ (not just in the brew file).
If you need the 2.2.0 part, then you can name it as `BASE_DEBEZIUM_VERSION`. Because, we added _our_ changes to that debezium version.